### DISCUSSION TITLE
Some PC cant boot by memtest

### DISCUSSION THREAD

**Discussion by changpenghui616** [memtest86plus/memtest86plus#189] [Q&A] [answered]
I create a bootable Memtest86+ USB Drive by Windows USB Installer.exe ，but display “boot fail” when i  choose boot from USB device in some computer（honor magicbook 16pro）.
PS：it can work in another PC
whether if this PC has some setting wrong ？
thanks.

**Comment by x86fr** [comment, marked as answer]
Did you checked that Secure Boot is disabled? As of today, Memtest86+ can't start with SecureBoot enabled.

**Reply by changpenghui616** [reply]
it work when disable secure boot，thanks.  :)
